src/components/FileUploader/FileUploader.tsx

```diff
@@ -6,52 +6,84 @@
  *
  */
 
-import { useState } from "react";
+import { Formik } from "formik";
+import { useMemo, useState } from "react";
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { spacing } from "@ndla/core";
-import { UploadDropZone } from "@ndla/forms";
+import { FileDocumentOutline } from "@ndla/icons/common";
+import { DeleteForever, UploadCloudLine } from "@ndla/icons/editor";
+import {
+  Button,
+  FieldErrorMessage,
+  FieldRoot,
+  FileUploadContext,
+  FileUploadDropzone,
+  FileUploadHiddenInput,
+  FileUploadItem,
+  FileUploadItemDeleteTrigger,
+  FileUploadItemGroup,
+  FileUploadItemName,
+  FileUploadItemPreview,
+  FileUploadItemSizeText,
+  FileUploadLabel,
+  FileUploadRoot,
+  FileUploadTrigger,
+  IconButton,
+  Text,
+} from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import { DRAFT_ADMIN_SCOPE } from "../../constants";
 import { useSession } from "../../containers/Session/SessionProvider";
 import { UnsavedFile } from "../../interfaces";
 import { uploadFile } from "../../modules/draft/draftApi";
 import { createFormData } from "../../util/formDataHelper";
 import handleError from "../../util/handleError";
 import { isNdlaErrorPayload } from "../../util/resolveJsonOrRejectWithError";
+import { FormField } from "../FormField";
+import { FormActionsContainer, FormikForm } from "../FormikForm";
+import validateFormik, { RulesType } from "../formikValidationSchema";
 
-const FileUploaderWrapper = styled.div`
-  padding: 0 ${spacing.large};
-`;
+const StyledErrorText = styled(Text, {
+  base: { color: "surface.danger", marginLeft: "auto" },
```

Comment:
Hm, lurer på om vi burde ha en token for `text.danger`

Reply:
Og bruk `marginInlineStart`!

Reply:
Enig!